README.md

```diff
@@ -460,10 +460,9 @@ db.createReadStream({ keys: false, values: true })
 <a name="writeStreams"></a>
 #### What happened to `db.createWriteStream`?
 
-Yes we have removed `db.createWriteStream` but not to worry, there is good reason for this. *Disclaimper*: if you are not in a stage where you are worried about performance but want a streaming interface
-into your database, please checkout [`level-ws`][level-ws].
+`db.createWriteStream()` has been removed in favor of a smaller core. The main reason for its existence was sake of symmetry with `db.createReadStream()`. For more information behind this move, please read the following [thread](https://github.com/rvagg/node-levelup/issues/199).
```

Comment:
Id rephrase this slightly...

`db.createWriteStream()` has been removed in order to provide a smaller and more maintainable core. It primarily existed to create symmetry with `db.createReadStream()` but through much [discussion](https://github.com/rvagg/node-levelup/issues/199), removing it was the best course of action. We encourage more `write-stream`' implementations to be created to solve the broad spectrum of use cases. 

Then we can lead into the paragraph below with something like.

Check out the implementations that the community has already produced [here](https://github.com/rvagg/node-levelup/wiki/Modules#write-streams)